### ISSUE TITLE
unloaded addon undefined

### ISSUE THREAD

**Issue by flatsiedatsie** [andrenatal/voice-addon#19]
I disabled the voice add-on (using the add-on menu), and got this in journalctl -f

```
Feb 18 13:32:47 gateway run-app.sh[559]: 2019-02-18 13:32:47.634 ERROR  : voice-addon: problem with request: connect ECONNREFUSED 127.0.0.1:4443
Feb 18 13:32:57 gateway run-app.sh[559]: 2019-02-18 13:32:57.636 ERROR  : voice-addon: problem with request: connect ECONNREFUSED 127.0.0.1:4443
Feb 18 13:33:07 gateway run-app.sh[559]: 2019-02-18 13:33:07.650 ERROR  : voice-addon: problem with request: connect ECONNREFUSED 127.0.0.1:4443
Feb 18 13:33:17 gateway run-app.sh[559]: 2019-02-18 13:33:17.654 ERROR  : voice-addon: problem with request: connect ECONNREFUSED 127.0.0.1:4443
Feb 18 13:33:21 gateway run-app.sh[559]: 2019-02-18 13:33:21.263 INFO   : Unloading VoiceAdapter
Feb 18 13:33:21 gateway run-app.sh[559]: 2019-02-18 13:33:21.269 INFO   : voice-addon: unloaded addon undefined
```

**Comment by andrenatal** [comment]
This was probably fixed on version 1.0.0. Would you mind please giving it a try @flatsiedatsie ?

**Comment by flatsiedatsie** [comment]
Was not fixed.
